### PR TITLE
release-velero: add changes in release-velero: disaster and recovery

### PR DESCRIPTION
**Pull request by cclhsu**
release-velero: add changes in release-velero: disaster and recovery for release notes
Signed-off-by: cclhsu <clark.hsu@suse.com>

### CODE REVIEW

File: adoc/MAIN.release-notes.adoc
Comment (jenting) [inline adoc/MAIN.release-notes.adoc]: S3 storage or S3-compatible storage.

File: adoc/MAIN.release-notes.adoc
Comment (c3y1huang) [inline adoc/MAIN.release-notes.adoc]:
```suggestion
You can now install `etcdctl`, the CLI to provide access to etcd cluster from the management host to backup and restore cluster data. For detailed instructions please see link: https://documentation.suse.com/suse-caasp/4.1/single-html/caasp-deployment/#_backup_etcd_cluster_data[the Deployment Guide].
```

**Review by c3y1huang — APPROVED** [review]

File: adoc/MAIN.release-notes.adoc
Comment (jenting) [inline adoc/MAIN.release-notes.adoc]: I cannot check the link is correct or not, but is it `#_disaster_Recovery` or `#_disaster_recovery` or ...?

Comment (cclhsu) [reply adoc/MAIN.release-notes.adoc]: the doc is not merge/publish yet. I just created the link following caption convention to link.

Comment (r0ckarong) [reply adoc/MAIN.release-notes.adoc]: You can typically check the state of `master` on https://susedoc.github.io/doc-caasp/master/single-html/caasp-admin/#_disaster_recovery once it's merged it takes a few minutes to be rebuild by the CI.

Then you just need to replace the first bit with the parts for the final URL.
`https://susedoc.github.io/doc-caasp/master/` with
`https://documentation.suse.com/suse-caasp/4.1/`. If the URL works on github.io then it should work in the final published document.

If you verify the docs locally (e.g. with the AsciiDoctorJS browser extension; recommended) then you can just look at the filename + anchor and interpolate the URL to one of these formats.

File: adoc/MAIN.release-notes.adoc
Comment (innobead) [inline adoc/MAIN.release-notes.adoc]: @cclhsu thanks a lot. It looks good to me, but I would like to highlight the outstanding features more to align with the documentation. Please refer to the below and adjust if need. Thanks.

```
=== Add CLI etcdctl

You can now install `etcdctl`, the CLI to provide access to etcd cluster to backup and restore etcd data. For detailed instructions please see link: https://documentation.suse.com/suse-caasp/4.1/single-html/caasp-admin/#_backup_etcd_cluster_data[the Administration Guide].

=== etcd Backup and Restore for Master Nodes Disaster Recovery

- Provide etcd backup process on-demand or by schedule to prevent etcd data corruption.
- Provide etcd restore process to recover failed master node(s) to restore etcd quorum for cluster serving. 

For detailed instructions please see link: https://documentation.suse.com/suse-caasp/4.1/single-html/caasp-admin/#_backup_etcd_cluster_data[the Administration Guide].

=== Velero for Disaster Recovery

- Provide Velero as a solution for data protection and data migration by backing up and migrating Kubernetes resources and persistent volumes to and from externally supported storage backend on demand or by schedule. 

For detailed instructions please see link: https://documentation.suse.com/suse-caasp/4.1/single-html/caasp-admin/#_disaster_recovery[the Administration Guide].
```

**Review by innobead — CHANGES_REQUESTED** [review]

**Comment by r0ckarong** [comment]
@innobead Please provide final review and then I will merge this after the coordination meeting. Thanks!

**Review by innobead — APPROVED** [review]

**Pull request merged**
